process-engine/process_engine_runtime
# ♻️ Move http blocker parameters from NodeEnv to "startRuntime" arguments

## Changes

1. Remove `NO_HTTP` and `DO_NOT_BLOCK_GLOBAL_ROUTES` environment parameters
2. Accept `enableHttp` and `useHttpRootRoutes` as optional parameters for `startRuntime`
    - In essence, these will do what the removed environment variables did, but with a default positive value
3. Update the Readme
4. Update the tests for the root http routes
5. Add missing `types` section to `package.json`

## Issues

Closes #461 

PR: #466

## How to test the changes

- The Node env parameters `NO_HTTP` and `DO_NOT_BLOCK_GLOBAL_ROUTES` should not work anymore
- The parameters for `startRuntime()` should now accept two additional optional parameters:
    - `enableHttp`: Enable/Disable all http endpoints. Defaults to `true`
    -  `useHttpRootRoutes`: Enable/Disable usage of `/` and `/security/authority` routes
